src/schema/objects/columns.yaml

```diff
@@ -122,6 +129,12 @@ description__optode:
   description: |
     Free-form text description of the optode, or other information of interest.
   type: string
+description__descriptions:
```

Comment:
If we choose to implement other entity definition TSVs (see #1706), I wonder if we'd want to use the same column name ("description") for the value definitions or if we'd want to use the full entity names (e.g., `acq_id` and `acquisition`). I think `acq_id` and `description` makes more sense TBH, in which case maybe the identifier here should be `description__entity`.

Reply:
Per chat with @effigies, `description__entity` makes more sense. We can customize the definitions as they appear in the specification text with the macros.